Explorer/Assets/DCL/SDKComponents/MediaStream/Components/MediaPlayerComponent.cs

```diff
@@ -10,39 +13,96 @@ public struct MediaPlayerComponent
         public const float DEFAULT_PLAYBACK_RATE = 1f;
         public const float DEFAULT_POSITION = 0f;
 
+        private float lastVideoTime;
+        private float frozenTimestamp;
+        private bool isFrozen;
+
         public readonly MultiMediaPlayer MediaPlayer;
         public readonly bool IsFromContentServer;
 
         public MediaAddress MediaAddress;
-
         public VideoState State { get; private set; }
+        public ErrorCode ErrorCode;
```

Comment:
Its a little bit strange thet you are using the AVPRO api here; shouldnt it be inside `MediaPLayerComponent`?

And if its shared with Livekit, shouldnt it be a different API?

Reply:
True. Done here: https://github.com/decentraland/unity-explorer/pull/6121/commits/dc1f01325c0f518850b03075fb18843fa5a24f59